Misc/NEWS.d/next/Security/2025-03-14-23-28-39.gh-issue-131261.0aB6nM.rst

```diff
@@ -0,0 +1 @@
+Upgrade to libexpat 2.7.0
```

Comment:
Should this mention the CVE?

Reply:
the previous 2.6.3 update didn't so I didn't here, but maybe?  no strong opinion myself.